buildSrc/src/main/kotlin/scripts/variants.gradle.kts

```diff
@@ -124,10 +131,17 @@ android {
             isMinifyEnabled = true
             proguardFiles(getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro")
             isDebuggable = false
-            matchingFallbacks.add("release")
```

Comment:
we need this line since kalium does not offer the same types as the app so it will not build for COMPAT_RELEASE